Refine the web hook and notify commit docs (#836)
jenkinsci/git-plugin: README.adoc

It does not support other credential types like secret text, secret file, or certificates.
Select credentials from the job definition drop down menu or enter their identifiers in Pipeline job definitions.

[[push-notification-from-repository]]
[[GitPlugin-Pushnotificationfromrepository]]
=== Push notification from repository

To minimize the delay between a push and a build, configure the remote repository to use a Webhook to notify Jenkins of changs to the repository.

> Refer to webhook documentation for your repository:

Author: Needs a separating blank line to recognize the list
```suggestion
Refer to webhook documentation for your repository:

```

It does not support other credential types like secret text, secret file, or certificates.
Select credentials from the job definition drop down menu or enter their identifiers in Pipeline job definitions.

[[push-notification-from-repository]]
[[GitPlugin-Pushnotificationfromrepository]]
=== Push notification from repository

To minimize the delay between a push and a build, configure the remote repository to use a Webhook to notify Jenkins of changs to the repository.
Refer to webhook documentation for your repository:

* link:https://plugins.jenkins.io/github#GitHubPlugin-GitHubhooktriggerforGITScmpolling[GitHub]
* link:https://plugins.jenkins.io/bitbucket[Bitbucket]
* link:https://plugins.jenkins.io/gitlab-branch-source[GitLab]
* link:https://github.com/jenkinsci/gitea-plugin/blob/master/docs/README.md[Gitea]

> Other git repositories can notify Jenkins with a post-receive hook in the repository to poke Jenkins when a new push occurs.

Author: Changes are more than `git push`.
```suggestion
Other git repositories can use a link:https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks[post-receive hook] in the remote repository to notify Jenkins of changes.
```

> Add the following line in your hooks/post-receive file on the git server, replacing <URL of the Git repository> with the fully qualified URL you use when cloning the repository.

Author: ```suggestion
Add the following line in your `hooks/post-receive` file on the git server, replacing <URL of the Git repository> with the fully qualified URL you use when cloning the repository.
```

> Add the following line in your hooks/post-receive file on the git server, replacing <URL of the Git repository> with the fully qualified URL you use when cloning the repository.

Author: ```suggestion
Add the following line in your `hooks/post-receive` file on the git server, replacing <URL of the Git repository> with the fully qualified URL you use when cloning the repository.
```

....
curl http://yourserver/git/notifyCommit?url=<URL of the Git repository>
....

This will scan all the jobs that:

* Have Build Triggers > Poll SCM enabled.  No polling schedule is required.
* Are configured to build the repository at the specified URL

For jobs that meet these conditions, polling will be triggered.
If polling finds a change worthy of a build, a build will be triggered.

This allows a script to remain the same when jobs come and go in Jenkins. 

> Or if you have multiple repositories under a single repository host application (such as Gitosis, gitweb, or ), you can share a single post-receive hook script with all the repositories.

Author: ```suggestion
Or if you have multiple repositories under a single repository host application (such as Gitosis), you can share a single post-receive hook script with all the repositories.
```
